pom.xml

```diff
@@ -76,10 +76,14 @@
         <module>appconfiguration/spring-cloud-azure-starter-appconfiguration/appconfiguration-client</module>
         <module>cache/spring-cloud-azure-starter/spring-cloud-azure-sample-cache</module>
         <module>cloudfoundry/azure-cloud-foundry-service-sample</module>
+        <module>cosmos/spring-cloud-azure-starter-data-cosmos/spring-cloud-azure-data-cosmos-aks-sample</module>
```

Comment:
for any future modules we add to our samples, do we need to add them to this pom.xml ?

Reply:
Yes.